js/plugins/tooltips.js

```diff
@@ -93,6 +93,12 @@ export function install(Vue) {
                 }
             }, popover));
 
+            // Transclude child content if found
+            const content = this.el.querySelector('[popover-content]');
```

Comment:
because I simply removed the `v-` prefix and used an attribute.
I can move it to a data-attribute if you prefer